docs/consensus/migration-to-pos.md

```diff
@@ -0,0 +1,20 @@
+---
+id: migration-to-pos
+title: Migration from PoA to PoS
+---
+
+## Overview
+
+This section guides you through migration from PoA to PoS for the running cluster without reset of blockchain.
+
+## How to migrate to PoS
+
+You will need to stop all nodes, update genesis.json, and restart the nodes in order to switch to PoS. `ibft switch` command append fork configuration into `genesis.json`.
+
+````bash
+polygon-edge ibft switch --chain ./genesis.json --type PoS --deployment 100 --from 200
+````
+
+To switch to PoS, you will need to specify 2 block heights: `deployment` and `from`. `deployment` is the height to deploy the staking contract and `from` is the height of beginning of PoS. The staking contract will be deployed at the address `0x0000000000000000000000000000000000001001`  at the `deployment`, like as the case of pre-deployed contract.
```

Comment:
Please add links to the Staking SC doc / repo here 🙏

Reply:
Fixed at c54b7deb2c83464f83a348a110ea278cef3f0798